test/Table_Tests/src/Common_Table_Operations/Add_Row_Number_Spec.enso

```diff
@@ -107,16 +125,24 @@ spec setup =
             t2.at "Sum Y" . to_vector . should_equal [5.0, 2.0, 8.0]
             t2.at "Row" . to_vector . should_equal [1, 2, 3]
 
-    if setup.is_database.not then Test.group prefix+"Table.add_row_number (in-memory specific)" <|
-        Test.specify "should add a row numbering column" <|
+    if setup.is_database.not then suite_builder.group prefix+"Table.add_row_number (in-memory specific)" group_builder->
+        data = Data.setup create_connection_fn
+
+        group_builder.teardown <|
+            data.teardown
+
+        table_builder cols =
+            setup.table_builder cols connection=data.connection
```

Comment:
This pattern ensures that any table created via `table_builder` was created from the same connection. Otherwise, tests with database backends would be failing on errors like "Tables were not created in the same connection".

Reply:
I think it would be useful to put this comment into the code, otherwise it will be forgotten.